tools/apiview/parsers/rust-api-parser/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.0.1
+Updated the parser to handle multi-line doc comments and `ReviewLine`s to improve rendering in the API View tool, due to a breaking change in the API View tool's handling of multi-line `ReviewLine`.
```

Comment:
It's not exactly a breaking change. No `ReviewLine` should span multiple lines. Generally the parser should enforce this.

Reply:
I termed it a breaking change since the behavior changed from the API View tool side.